aip/cloud/2510.md

```diff
@@ -49,9 +49,14 @@ However, the project number is the canonical identifier, and therefore APIs
 **must** use project numbers in responses, regardless of whether they were sent
 a project ID or a project number.
 
-An exception is for error responses, which **must** return the
-originally-provided value without modification. Error responses **must not**
-perform any translation between project IDs and project numbers.
+There are two exceptions to this rule:
+
+- Error responses **must** return the originally-provided value without
+  modification. Error responses **must not** perform any translation between
+  project IDs and project numbers.
+- If a service is receiving a resource name for a resource that the service
```

Comment:
Personally I'd change "is receiving" to "receives" but I'm not overly bothered

Reply:
```suggestion
- If a service receives a resource name for a resource that the service
```